psalm.baseline.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="3.11.5@3c60609c218d4d4b3b257728b8089094e5c6c6c2">
+<files psalm-version="4.x-dev@">
```

Comment:
I'm curious as to why this now indicates a development branch rather than a specific release version.

Reply:
This line (actually, the complete file) is autogenerated. It looks like the Psalm project has changed their `psalm-version` schema.